docs/mkdocs.yml

```diff
@@ -38,6 +38,7 @@ nav:
           - Contacts: Contribution guidelines/security/security_contacts.md
       - Operations: 
           - Code Management: Contribution guidelines/operations/code-management.md
+          - Versioned Things: Contribution guidelines/operations/versioned-things.md
```

Comment:
how about "Notes on Versioning"

Reply:
I think the most important thing here is not the versions but just the survey of different sorts of piles of stuff that we have and how they are related to each other. The versioning will help with consistency when navigating those relationships. I also was thinking, as I drew the picture, that it might be a good idea to add the next layer of context, users and Helm charts that invoke ours and ... well, I am not sure what all.

I think perhaps "packaging" is a word that conveys what I am trying to say.  What about a title like "Packaging and Versioning"?

Reply:
renamed as I suggested.